web/src/pages/people/index.tsx

```diff
@@ -45,6 +55,30 @@ export default function PeoplePage() {
   const isIAO = userData?.roles.includes("information-asset-owner");
   const isTreOpsStaff = userData?.roles.includes("tre-ops-staff");
 
+  const handleUserSearch = async (query: string) => {
+    if (query !== "") {
+      const response = await getUsers({ query: { find: query } });
+      if (response.response.ok && response.data) {
+        setUsers(response.data);
+      } else {
+        setSearchErrorMessage("Oops, something went wrong with your search. Refresh and try again");
+      }
+    } else {
+      setUsers(originalUsers);
+    }
+  };
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(event.target.value);
+    if (event.target.value === "") {
+      setUsers(originalUsers);
+    }
+  };
+  const handleUserSearchSubmit = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    handleUserSearch(searchTerm);
+  };
```

Comment:
Is this additional abstraction needed? Couldn't the logic all go in `handleUserSearch`?

Reply:
putting the code into the `handleUserSearch` function and then calling it from the form or button makes things unhappy due to expected types of events, I think it's cleaner and easier to read as separate things but if you have a code suggestion I'm open

Reply:
🤦‍♀️ 
forgot we could do it that way